src/scripts/main.js

```diff
@@ -1,5 +1,36 @@
 'use strict';
 
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
 document.addEventListener('click', (e) => {
   // write code here
+  const wallCoords = wall.getBoundingClientRect();
+  const spiderCoords = spider.getBoundingClientRect();
+  const border = parseInt(
+    window.getComputedStyle(wall).borderWidth.split('px').join(''),
```

Comment:
Consider using `parseFloat` instead of `parseInt` when parsing the border width. This will handle cases where the border width might be a decimal value.